main.js

```diff
@@ -0,0 +1,14 @@
+let allPosts = document.querySelector("header nav ul li");
+allPosts.remove();
```

Comment:
```suggestion
let firstLi = document.querySelector("header nav ul li");
firstLi.remove();
```
I just changed the variable name from `allPosts` to `firstLi` because `document.querySelector` always grabs a **single** element: the **first** element that matches the selector (in this case "header nav ul li"). `allPosts` suggest multiple elements, but I want it to be clear that this is a single element.